_rules/80f0bf.md

```diff
@@ -0,0 +1,112 @@
+---
```

Comment:
change filename to something of the format someText + uniqueId eg: `no-auto-play-audio-80f0bf.md`